exporter/sumologicexporter/carbon_formatter.go

```diff
@@ -0,0 +1,108 @@
+// Copyright 2021, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sumologicexporter
+
+import (
+	"fmt"
+	"strings"
+
+	"go.opentelemetry.io/collector/consumer/pdata"
+	tracetranslator "go.opentelemetry.io/collector/translator/trace"
+)
+
+// carbon2TagString returns all attributes as space spearated key=value pairs
+// In addition, metric name and unit are also included
+// In case `metric` or `unit` attributes has been set too, they are prefixed with underscore `_`
```

Comment:
Is this prefix required for something? Or is it required by carbon2 spec? I'd add that note here.

Reply:
It's not in the carbon2 spec, carbon2 spec doesn't refer to any implementation (like otc metric structure)

Reply:
So what's the reason this is being done?

Reply:
because of using metric structure you can have both metric name and attribute which is named `name` and I want to keep both of them